### ISSUE TITLE
<domain>/bitbucket/<project> does not work

### ISSUE THREAD

**Issue by mskyaxl** [tommcdo/vim-fubitive#25]
Hi,

Really cool project but I have some issues getting it working for a bitbucket domain that is ``code.example.com/bitbucket/``

My remote is:

``ssh://git@code.example.com:<port>/<SomeProject>/<someRepo>.git ``

and the bitbucket link is 

``https://code.example.com/bitbucket/projects/<SomeProject>/repos/<someRepo>/<filepath>``

I tried with
```
let g:fubitive_domain_pattern = 'code\.example\.com'
```
but I don't know if there is a config to add ``bitbucket`` at the end of the domain.

I had a look over the source code and I could not find a way to change it through a config. My vimscript knowledge is limited though.

Just to test it out, adding an additional configurable variable "fubitive_domain_context_path" that is by default initialized with "". The value of this variable would be concatenated on the line https://github.com/tommcdo/vim-fubitive/blob/master/plugin/fubitive.vim#L36. This worked for me.

This would allow users to have the following configuration to cover my usecase:

```
let g:fubitive_domain_pattern = 'code\.example\.com'
let g:fubitive_domain_context_path = 'bitbucket'
```
Did I miss something? Is there any other way to cover my usecase?
Should I create a PR with this?

**Comment by mark2185** [comment]
How common is this across companies/projects? I'm just curious.

I mean, we _could_ add it since it's a non-breaking feature. I just hope no-one will come with a totally different pattern in the future :)

I'm open for a PR!

**Comment by mskyaxl** [comment]
I don't know how common is but in my current company but also in the previous one we had context paths like. <domain>/jira, <domain>/bitbucket, etc.

**Comment by mark2185** [comment]
Solved by #31.
